app/src/NativeBase/Containers/MyExperienceContainer.tsx

```diff
@@ -0,0 +1,141 @@
+/**
+ * @file the MyExperience screen for the STA Volunteer App
+ */
+import {
+  Heading,
+  VStack,
+  HStack,
+  ScrollView,
+  Flex,
+  View,
+  Text,
+} from 'native-base'
+import ProfileButtons from '../Components/ProfileButtons'
+import ProgressBar from '../Components/ProgressBar'
+import StaThemeLogo from '@/NativeBase/Assets/Images/Logos/sta-ribbon-logo.svg'
+import FreeSearchBar from '../Components/FreeSearchBar'
+import React from 'react'
+import Checkbox from '../Components/Checkbox'
+import { RoleGroup, roleGroups } from '@/Services/modules/projects/roleGroups'
+import {
+  getProgressBarColors,
+  nextScreen,
+} from '../../Utils/ProgressBarColours'
+
+const MyExperienceContainer = () => {
+  const [skillsValue, setSkillsValue] = React.useState<string[]>([]) // skills selected by the user
+  const [searchTxt, setSearchTxt] = React.useState<string>('') // search text
+  const [currentBox, setCurrentBox] = React.useState<number>(1) // current box in the progress bar
+
+  /**
+   * Filter the role groups based on the search text entered by the user
+   */
+  const filteredSkills = roleGroups.filter((roleGroup: RoleGroup) =>
+    roleGroup.groupName.toLowerCase().includes(searchTxt.toLowerCase()),
+  )
+
+  //  get the progress bar colours for the current box in the progress bar
+  const progressBarColourObject = Object.values(
+    getProgressBarColors(currentBox, 'purple.20'),
+  )
+
+  return (
+    <>
+      <HStack mt={-10} mb={5} alignItems={'flex-end'}>
+        <Heading size="md" fontFamily="title" adjustsFontSizeToFit>
+          My Experience
+        </Heading>
+        <Flex flex={1} mt={-5} alignItems={'flex-end'}>
+          <StaThemeLogo />
+        </Flex>
+      </HStack>
+      <ProgressBar colours={progressBarColourObject} />
+      <View margin={'4'}>
```

Comment:
Tiny thing, but there are some minor inconsistencies in how values are being set -- it's not entirely clear from the NativeBase docs, but it looks like for number settings either `margin="4"` or `margin={4}` are both valid